src/test/java/org/cactoos/iterable/MatchedTest.java

```diff
@@ -56,11 +57,13 @@ public void iterator() {
     public void noCorrelationWithBiggerSecondIterable() {
         new Assertion<>(
             "All elements have correlation function as `endsWith`",
```

Comment:
@andreoss Please, change ```  for `'`.

Reply:
@fabriciofx Do you mean in general or in the tests cases?

Reply:
@andreoss In general